src/main/java/com/sarahand/fractals/json/ConfigSaverLoader.java

```
@@ -11,50 +12,63 @@
 
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 import com.sarahhand.fractals.model.Config;
+import com.sarahhand.fractals.model.MandelbrotConfig;
 import com.sarahhand.json.JsonReaderWriter;
 
 public class ConfigSaverLoader{
-	
+
 	private final Logger log = LoggerFactory.getLogger(this.getClass());
-	
+
 	private static final ConfigSaverLoader DEFAULT = new ConfigSaverLoader();
-	
+
 	private ConfigSaverLoader(){}
-	
+
 	public static ConfigSaverLoader getDefaultConfigSaverLoader(){
 		return DEFAULT;
 	}
-	
-	public Config load(Class<? extends Config> target, InputStream stream){
-		
+
+	public Config load(Config config, String fileName){
+
 		JsonReaderWriter readerWriter = new JsonReaderWriter(PropertyNamingStrategy.SNAKE_CASE);
+
+		try(InputStream fis = new FileInputStream(fileName);) {
+			Config newConfig = readerWriter.<Config>read(fis, MandelbrotConfig.class);
+			return newConfig;
+		} catch(Exception e) {}
```

Add a logger to this class and log an error in the catch clause here.  Except in specific scenarios, it is considered poor programming practice to have an empty catch clause.

---

Done!